src/index.ts

```diff
@@ -1,14 +1,28 @@
 import { Command } from 'commander';
+import CodeforcesClient from '@acmascis/codeforces-client';
 
-const program = new Command();
+import { GetAllProblems } from './services/problems';
+import { getStatusData } from './services/status';
+import { generateContestXML } from './xmlGenerator/contestXML';
+import { GetAwards } from './services/awards';
+import { WriteToXmlFile } from './utils/fileManager';
 
-program
-	.requiredOption('--contest <contest-id>', 'cf contest id')
-	.requiredOption('--api-key <api-key>', 'cf api key');
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
[NITPICK] You can avoid this by not assigning it to a variable. Actually it doesn't make sense to make it an immediately-invoked function when you're assigning a name to it.

```js
// either
(async () => {})();

// or
const main = async () => {}
main();
```